README.md

```diff
@@ -19,60 +19,120 @@ GeoMesa is an open-source, distributed, spatio-temporal database built on top of
 
 ## Download and Version Information
 
-<b>NOTE:</b> The current recommended version is ```1.1.0-rc1```, which includes breaking changes from ```1.0.0-rc7``` and previous versions due to schema incompatibility and changes in indexing structure. The most recent tar.gz assembly can be 
-[downloaded here](http://repo.locationtech.org/content/repositories/geomesa-releases/org/locationtech/geomesa/geomesa-assemble/1.1.0-rc.1/geomesa-assemble-1.1.0-rc.1-bin.tar.gz) which contains the [Accumulo distributed runtime jar](geomesa-distributed-runtime), [GeoServer plugin](geomesa-plugin), and [command line tools](geomesa-tools).
+<b>NOTE:</b> The current recommended version is ```1.1.0-rc.6```, which includes breaking changes from ```1.0.0-rc.7``` and previous versions due to schema incompatibility and changes in indexing structure. The most recent tar.gz assembly can be 
```

Comment:
somewhere we do need to mention that even upgrading between "minor rc" releases that you must upgrade both the tserver and then geoserver plugin for it to work (at least for rc.6 to rc.7-snapshot you do)_